src/chi-vue/src/components/column-customization/SelectedColumns.tsx

```diff
@@ -37,15 +41,31 @@ export default class ColumnCustomizationSelectedColumns extends Vue {
     }
   }
 
+  changeSelectRowsState(column: DataTableColumn) {
+    const selectButton = document.querySelector('.icon-chevron-right')?.parentElement
+      ?.parentElement as HTMLButtonElement;
+    const deselectButton = document.querySelector('.icon-chevron-left')?.parentElement
+      ?.parentElement as HTMLButtonElement;
+
+    if (selectButton && deselectButton) {
+      if (column.locked) {
+        selectButton.disabled = true;
+        deselectButton.disabled = true;
+        return;
+      }
+    }
+
+    selectButton.disabled = false;
+    deselectButton.disabled = false;
```

Comment:
Here you are selecting the element and changing their state. Instead, when generating these components, you need to set
disabled = {CONDITION_OF_DISABLING}
and when selecting options in selected columns list, change the CONDITION_OF_DISABLING so that the button renders in the proper state.

Reply:
Solved.